working-days/working-days.component.html

```diff
@@ -0,0 +1,66 @@
+<div class="container">
+  <form>
+    <table style="border:0px">
+        <h4>Working Days</h4><span style="padding: 40px"></span>
+      <div class="container" fxLayout="coloumn" fxFlex="80" fxLayoutAlign="left">
+        <tr>
+          <mat-checkbox color="primary">Monday</mat-checkbox>
+        </tr>
+        <tr>
+          <mat-checkbox color="primary">Tuesday</mat-checkbox>
+        </tr>
+        <tr>
+          <mat-checkbox color="primary">Wednesday</mat-checkbox>
+        </tr>
+        <tr>
+          <mat-checkbox color="primary">Thursday</mat-checkbox>
+        </tr>
+        <tr>
+          <mat-checkbox color="primary">Friday</mat-checkbox>
+        </tr>
+        <tr>
+          <mat-checkbox color="primary">Saturday</mat-checkbox>
+        </tr>
+        <tr>
+          <mat-checkbox color="primary">Sunday</mat-checkbox>
+        </tr>
+      </div>
+    </table>
+  </form>   
+</div>
+
+<div class="container" fxLayoutAlign="left">
+  <form>
+    <table style="border:0px">
```

Comment:
Do not use tables for each field. Use a mat-form-field with mat-label instead to ensure consistency. Please see src/app/accounting-rules/create-rule/create-rule.html for reference.

Reply:
Adding on to this please don't use any inline styles as well. The `scss` file for the component is missing which should be used to implement the styles.